go/vt/graph/graph.go

```diff
@@ -18,19 +18,28 @@ package graph
 
 import (
 	"fmt"
+	"maps"
 	"slices"
 	"strings"
 )
 
+const (
+	white int = iota
+	grey
+	black
+)
+
 // Graph is a generic graph implementation.
 type Graph[C comparable] struct {
-	edges map[C][]C
+	edges        map[C][]C
+	orderedEdged []C
```

Comment:
This should be named `orderedVertices` instead of `orderedEdged` right? C represents a vertex.

Reply:
Right! Renamed.